addons/web/static/src/views/list/list_renderer.scss

```diff
@@ -176,10 +176,6 @@
             cursor: pointer;
         }
 
-        .o_list_record_remove button, button {
```

Comment:
:warning: This line has *2 selectors* : `.o_list_record_remove button` and the super generic `button` (even if scoped in a list view's table) !

Reply:
Yes the whole line was added in https://github.com/odoo/odoo/commit/258e6a019a21042bf4f6cf70fcce386d37afd50c, it's deleted on purpose else it would misalign any other button rendered in the list view (also the reason why the trashbin was misaligned). Is it causing anything unwanted?

Reply:
No, not that I know of.
As it was easy to miss, I preferred to note it. :grin: